lib/web/apiserver.go

```diff
@@ -894,10 +894,22 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 		PreferredLocalMFA: cap.GetPreferredLocalMFA(),
 	}
 
+	// get tunnel address to display on cloud instances
+	tunnelPublicAddr := ""
+	if h.ClusterFeatures.GetCloud() {
+		proxyConfig, err := h.cfg.ProxySettings.GetProxySettings(r.Context())
+		if err != nil {
+			h.log.WithError(err).Error("Cannot retrieve ProxySettings.")
```

Comment:
`"Cannot retrieve ProxySettings."` is just logged (not returned as function error) in this case. Someone reading logs probably won't know what is going on. As this is not "fatal" error can we add something like `tunnel address won't be set in Web UI` to make it more clear and consider making this a warning?

Reply:
Thanks @jakule , this makes sense
I've changed to a to a warning and added `tunnel address won't be set in Web UI`